docs/en/engines/table-engines/special/distributed.md

```diff
@@ -146,5 +146,6 @@ When the max\_parallel\_replicas option is enabled, query processing is parallel
 **See Also**
 
 -   [Virtual columns](index.md#table_engines-virtual_columns)
+-   [background_schedule_pool_size](../../../operations/settings/settings.md#background_schedule_pool_size)
```

Comment:
1. The setting for this case is `background_distributed_schedule_pool_size`.
2. A better place for this link is the paragraph "Data is written asynchronously..."